custom_components/zwave_mqtt/__init__.py

```diff
@@ -246,6 +250,35 @@ async def handle_remove_node(hass: HomeAssistant, node_id: int):
             dev_registry.async_remove_device(dev_id)
 
 
+async def handle_node_update(hass: HomeAssistant, node: OZWNode):
+    """
+    Handle a node updated event from OZW.
+
+    Meaning some of the basic info like name/model is updated.
+    We want these changes to be pushed to the device registry.
+    """
+    dev_registry = await get_dev_reg(hass)
+    # grab device in device registry attached to this node
+    device = dev_registry.async_get_device([(DOMAIN, node.id)], [])
+    if device:
+        # update device in device registry with (updated) info
+        # also handle any slave devices (node instances)
+        for item in dev_registry.devices.values():
+            if item.id == device.id or item.via_device_id == device.id:
+                dev_name = f"{node.node_manufacturer_name} {node.node_product_name}"
+                if item.via_device_id:
+                    instance = list(item.identifiers)[0][2]
+                    dev_name += f" - Instance {instance}"
+                # TEMP: Using private update method as public uodate needs extending.
+                # TODO: Use public function as soon as core is adjusted,
```

Comment:
This has been added https://github.com/home-assistant/core/pull/34500

Reply:
Nice, that's quick :-)

Reply:
What to do ? Adjust code and not merge until a new hass version comes out ?
Or merge like it is now and insert a separate PR at the same time to adjust the private to pubic method ?

As we're publishing this into HACS we need to make sure we do not publish code depending om core changes

Reply:
How does the HACS publish work. Does it use current master branch or latest github release?

Reply:
latest GH release. I'd like to do a new release today or tomorrow with yesterday's fixes (and the PR I'm about to do)

Reply:
In hacs.json in the root of the repo you can add a minimum home assistant version: `"homeassistant": "0.109.0",`

Reply:
What happens if that home assistant version isn't released yet? Does the HACS integration not install then? Can you select different HACS integration versions?

Reply:
Not sure how it will handle an HA version that hasn't been released yet. @ludeeus ?

Reply:
It will tell the user to upgrade

Reply:
Should we require 0.109.0, then merge this and wait to release another version here until 0.109.0 is released on Wednesday?

Reply:
Does 0.109 contain the fix ? In that case we'll wait with merging, that's only a few days from now.

Reply:
Yeah, the fix is in rc branch.

Reply:
Code adjusted to use the public function (released in Hass 0.109). We should only make sure that 0.109 is set as minimum version for next release on HACS.